core/prefs-macros.h

```diff
@@ -18,6 +18,13 @@
 	else                                                        \
 		prefs.units.field = default_prefs.units.field
 
+#define GET_UNIT_INT(name, field)             \
+	v = s.value(QString(name));      \
+	if (v.isValid())                 \
+		prefs.units.field = v.toInt(); \
+	else                             \
+		prefs.units.field = default_prefs.units.field
+
```

Comment:
best to write this as:
```
v = s.value(QString(name)); \
prefs.units.field = v.isValid() ? v.toInt() : default_prefs.units.field \
```

Reply:
This is copy&paste from a couple of other macros which look the same syntax wise.
So either I leave it as it is or change all macros in the same way but this would be another commit (and a pure "make the syntax nice thing"). Honestly I prefer to leave it :-)

Reply:
ok, i guess leave as is for now.